### ISSUE TITLE
Interaction with monster-card

### ISSUE THREAD

**Issue by OppyLock** [ljmerza/travel-time-card#3]
I was hoping to get this card to auto-populate with a filtered selection of sensors.  I currently use an entities card to show potential active routes, like this:

card:
..show_header_toggle: false
..title: Travel Times
..type: entities
filter:
..include:
....- attributes:
........duration: '> 2'
........domain: sensor
show_empty: false
type: 'custom:monster-card'

This gets me a card showing the travel times of potential trips.  If I change line 4 to type: 'custom:travel-time-card' then I get nothing at all in lovelace.  The travel-time-card works if I manually populate the entities though.

**Comment by ljmerza** [comment]
Are you asking for support for a filter option?

**Comment by ljmerza** [comment]
just leaving a note here for myself

```
- card:
          show_header_toggle: false
          title: Travel Times
          type: custom-travel-time-card
        filter:
          include:
            - attributes:
                duration:  '<= 0'
        type: 'custom:monster-card'
```

**Comment by OppyLock** [comment]
The idea of the monster card is that it does the dynamic filtering and punts the resulting entities up to the next card in the chain.  It would save implementing any filtering code in your card.  But, for some reason, the travel card doesn't seem to accept the entities thus provided.

**Comment by ljmerza** [comment]
I looked into the code for the monster card and I dont see any type of support ofr custom cards. The card is filtering correctly but when it tries to create the child card it does this:

```const element = document.createElement(`hui-${cardConfig.card.type}-card`);```

With your configuration it tries to find the `hui-custom:travel-time-card-card` which obviously doesnt exist. Even with passing only `travel-time` the moster card will try to find `hui-travel-time-card` which doesnt exist. I tried to override the code to see if it would make the card but it still didn't. I would guess this is an issue of custom cards not loading before this card.

Either way, I'm pretty sure by looking at the code, mostner card doesn't support custom cards.
